### PR TITLE
feat: enableTcc, enableRemb, minParticipants.

### PR DESCRIPTION
**Pull request by gpolitis**
Depends on https://github.com/jitsi/lib-jitsi-meet/pull/730

### CODE REVIEW

**Comment by gpolitis** [comment]
@paweldomas could you PTAL?

File: src/main/java/org/jitsi/jicofo/ParticipantChannelAllocator.java [hunk 1]
```diff
@@ -103,14 +105,16 @@ public ParticipantChannelAllocator(
         {
             contents.add(
                 jingleOfferFactory.createAudioContent(
-                    disableIce, useDtls, config.stereoEnabled()));
+                    disableIce, useDtls, config.stereoEnabled(),
+                    config.isRembEnabled(), config.isTccEnabled()));
```
Comment (bgrozev) [inline src/main/java/org/jitsi/jicofo/ParticipantChannelAllocator.java]: Why not use the local variables you defined above?

File: src/main/java/org/jitsi/jicofo/OctoChannelAllocator.java [hunk 2]
```diff
@@ -89,11 +91,12 @@ public OctoChannelAllocator(
         List<ContentPacketExtension> contents = new ArrayList<>();
         contents.add(
             jingleOfferFactory.createAudioContent(
-                    !useIce, useDtls, config.stereoEnabled()));
+                    !useIce, useDtls, config.stereoEnabled(),
```
Comment (bgrozev) [inline src/main/java/org/jitsi/jicofo/OctoChannelAllocator.java]: The content for audio uses the config, but the config for video is always false?

File: src/main/java/org/jitsi/jicofo/JitsiMeetConfig.java [hunk 3]
```diff
@@ -124,6 +130,12 @@
      */
     public static final int START_BITRATE_DEFAULT = 800;
 
+    /**
+     * The default value for the "minParticipants" property.
+     */
+    private static final int MIN_PARTICIPANTS_DEFAULT = 2;
```
Comment (paweldomas) [inline src/main/java/org/jitsi/jicofo/JitsiMeetConfig.java]: It is starting to get messy with more constants being added to this file. Some of them have PNAME, or DEFAULT suffix, some of them don't. Do you mind cleaning this up before things get out of hands ? What do you think about renaming all properties to PNAME_{property name} and DEFAULT_{property name} ? Then if we sort things alphabetically properties will be grouped with properties and defaults with defaults. If that's too much work or you don't like it, could you please at least sort all constants alphabetically ? This way defaults will be close to the properties.

Comment (gpolitis) [reply src/main/java/org/jitsi/jicofo/JitsiMeetConfig.java]: 👍

File: src/main/java/org/jitsi/jicofo/OctoChannelAllocator.java (hunk 2 above)
Comment (paweldomas) [reply src/main/java/org/jitsi/jicofo/OctoChannelAllocator.java]: @gpolitis is this now fixed ? I don't see such issue in the code anymore, am I missing something ?

File: src/main/java/org/jitsi/jicofo/JitsiMeetConfig.java (hunk 3 above)
Comment (paweldomas) [reply src/main/java/org/jitsi/jicofo/JitsiMeetConfig.java]: thanks !

File: src/main/java/org/jitsi/jicofo/OctoChannelAllocator.java (hunk 2 above)
Comment (gpolitis) [reply src/main/java/org/jitsi/jicofo/OctoChannelAllocator.java]: yes, now it's fixed

**Comment by paweldomas** [comment]
I know @bgrozev does not like force push, but the first commit is kind of unrelated, so I don't want to squash everything when merging. Is it ok to force push in this case Boris ?

**Comment by gpolitis** [comment]
I can squash the commits with the same title, with @bgrozev approval.

**Comment by bgrozev** [comment]
I don't mind a force push

**Comment by gpolitis** [comment]
I re-arranged the commits, please either approve (and I'll merge later) or approve & merge.

**Comment by bgrozev** [comment]
COMPILATION ERROR : 
22:11:27 [INFO] -------------------------------------------------------------
22:11:27 [ERROR] /home/jenkins/jicofo/src/test/java/org/jitsi/jicofo/ColibriThreadingTest.java:[323,40] method createAudioContent in class org.jitsi.jicofo.util.JingleOfferFactory cannot be applied to given types;

**Comment by gpolitis** [comment]
@paweldomas or @bgrozev could you please re-approve the final version? I re-based and fixed the compilation error.

**Pull request merged**